flow/network.h

```diff
@@ -91,6 +91,7 @@ enum class TaskPriority {
 	UnknownEndpoint = 4000,
 	MoveKeys = 3550,
 	DataDistributionLaunch = 3530,
+	BlobManager = 3520,
```

Comment:
We probably want this to be lower priority than DD/RK. Maybe 3490 or something?

Reply:
I'm not an expert on priorities though, so we probably want to run this by evan

Reply:
Yeah my decision was pretty arbitrary. I forgot to leave a TODO here to rethink this.
@sfc-gh-etschannen any suggestions?

Reply:
Evan agrees
> if we are saturated and have to pick what to work on, we probably want to do the other jobs over blob manager work

Changing to `3490`.